### PR TITLE
split mainframe

### PR DESCRIPTION
**Pull request by a29paul**
yeah I split it up

### CODE REVIEW

**Comment by 1298se** [comment]
Resolve #51

**Comment by 1298se** [comment]
Resolve #32

File: contrail-ui/src/components/main/mainStyles.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { createStyles, Theme} from "@material-ui/core/styles";
```
Comment (virajg224) [inline contrail-ui/src/components/main/mainStyles.ts]: spacing

Comment (1298se) [reply contrail-ui/src/components/main/mainStyles.ts]: Done

File: contrail-ui/src/components/main/MainFrame.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React, {Component} from "react";
+import MainAppBar from "./main-app-bar/MainAppBar";
+import MainDrawer from "./main-drawer/MainDrawer";
+import MainView from "./main-view/MainView";
+
+class MainFrame extends Component {
+    public render() {
+        return (
+            <div>
```
Comment (notoriousmango) [inline contrail-ui/src/components/main/MainFrame.tsx]: Does this need to be in div?

File: contrail-ui/src/components/main/main-app-bar/MainAppBar.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import { withStyles } from "@material-ui/styles";
+import React, { Component } from "react";
+import styles from "../mainStyles";
+
+class MainAppBar extends Component<any, any> {
+    public render() {
+        const { classes } = this.props;
+        return (
+            <body>
+                <div className={classes.root}>
```
Comment (notoriousmango) [inline contrail-ui/src/components/main/main-app-bar/MainAppBar.tsx]: Remove body tag. I think we already have our body tag. If I'm wrong keep it

Comment (1298se) [reply contrail-ui/src/components/main/main-app-bar/MainAppBar.tsx]: Fixed

File: contrail-ui/src/components/main/MainFrame.tsx (hunk 2 above)
Comment (1298se) [reply contrail-ui/src/components/main/MainFrame.tsx]: Yes it does, but it's better to put in a React Fragment. I changed

**Pull request merged**